index.html

```diff
@@ -4,15 +4,42 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>AJAX Project</title>
-  <link rel="icon" href="images/favicon.ico">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+    integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Exo&display=swap" rel="stylesheet" type="text/css" />
 </head>
 <body>
-  <header>
+  <header id="gray-background">
+    <div class="row">
+      <div class="column-full space-between">
+        <div class="logo">
+          <h1 class="header-text desktop">KEYWORD RELIEF</h1>
+          <h1 class="header-text mobile">KR</h1>
```

Comment:
For accessibility concerns, you shouldn't have multiple `h1` elements. You can only ever have 1 `h1`. The second one should become a `h2`, and you can just adjust your `font-size` to match the other `h1`